elements/designer-element-highlight/designer-element-highlight.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+
+<html>
+  <head>
+    <link rel="import" href="../../../polymer/polymer.html">
+  </head>
+  <body>
+
+    <dom-module id="designer-element-highlight">
+      <style>
+        :host {
+          display: block;
+          position: absolute;
+          box-sizing: border-box;
+        }
+        :host.before {
```

Comment:
before and after correspond to `insertBefore` / `insertAfter`, though the classes here are likely to change to top, left, bottom and right very soon to let us draw better insertion markers for inline and inline-block elements.